src/MongoDB.AspNetCore.OData/MongoDB.AspNetCore.OData.csproj

```diff
@@ -0,0 +1,13 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework>net5.0</TargetFramework>
```

Comment:
Do we need to support other/older frameworks? Net472?

Reply:
We cannot add support for any Framework versions as Microsoft.AspNetCore.OData package compatible with net core only. Regarding the core versions - the earliest we could support is .NETCoreApp 3.1 (also driven by OData library), but this version is out of support by Microsoft as of December 13, 2022. @JamesKovacs do you think we have to add support of it?